src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactFiberErrorLogger
+ * @flow
+ */
+
+'use strict';
+
+import type { CapturedError } from 'ReactFiberScheduler';
+
+function logCapturedError(capturedError : CapturedError) : void {
+  if (__DEV__) {
+    // console.log rather than console.error to avoid breaking tests
+    // (Jest complains about unexpected console.error calls.)
```

Comment:
The problem is that tests (at least ones I spot-checked) are already asserting that errors occurred (eg spying on `console.error` and asserting a specific number of calls). This increases the number though and so the tests failed.

I wanted to limit the size of this commit. But I'd be happy to update the 20-30 tests if you guys have a strong preference. 😄